renovate.json

```diff
@@ -17,7 +17,12 @@
     "enabled": true
   },
   "masterIssue": true,
+  "schedule": ["every weekend"],
   "packageRules": [
+    {
+      "packageNames": ["cypress"],
+      "extends": ["every weekday"]
```

Comment:
hmm, is this valid syntax? it is weird to extend a schedule. I would think something like

```
{
  "packageNames": ["cypress"],
  "schedule": "every weekday"
}
```

Reply:
Yes, per their docs
![Screen Shot 2020-09-09 at 4 05 32 PM](https://user-images.githubusercontent.com/21967/92655012-55763000-f2b6-11ea-8c15-784abb6a7640.png)

Reply:
Good catch

Reply:
Fixed per https://docs.renovatebot.com/presets-schedule/#scheduleweekdays